### ISSUE TITLE
Coverage does not work

### ISSUE THREAD

**Issue by liucijus** [bazelbuild/intellij#1179]
Running tests with Coverage does not work. Reproducable on other bazel projects.

Bazel plugin Version 2019.08.19.0.6
IntelliJ IDEA 2019.2.2 (Community Edition)
```
Testing started at 15:02 ...
/usr/bin/setsid --wait bazel coverage --tool_tag=ijwb:IDEA:community --curses=no --color=yes --progress_in_terminal_title=no --combined_report=lcov --define=ij_product=intellij-latest --runs_per_test=1 --flaky_test_attempts=1 --build_event_binary_file=/tmp/intellij-bep-4077e154-5aad-458f-b543-2ade2fa12228 --nobuild_event_binary_file_path_conversion --test_filter=com.google.idea.blaze.scala.run.producers.DeployableJarRunConfigurationProducerTest# -- //scala:integration_tests
Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
INFO: Using default value for --instrumentation_filter: "//scala[/:]".
INFO: Override the above default with --instrumentation_filter
Analyzing: target //scala:integration_tests (1 packages loaded, 0 targets configured)
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:486)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:429)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:198)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:110)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:141)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:268)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:133)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:113)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:518)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:192)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:573)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:624)
	at java.base/java.lang.Thread.run(Unknown Source)

INFO: Elapsed time: 3.436s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (64 packages loaded, 1891 targets configured)
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:486)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:429)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:198)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:110)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:141)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:268)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:133)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:113)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:518)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:192)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:573)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:624)
	at java.base/java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:486)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:429)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:198)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:110)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:141)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:268)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:133)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:113)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:518)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:192)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:573)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:624)
	at java.base/java.lang.Thread.run(Unknown Source)
FAILED: Build did NOT complete successfully (64 packages loaded, 1891 targets configured)
INFO: Build Event Protocol files produced successfully.
FAILED: Build did NOT complete successfully (64 packages loaded, 1891 targets configured)

Internal Bazel error. No tests were run.

```

**Comment by drichelson** [comment]
It would be really great to get coverage working. This is still failing for me in some recent versions:
IntelliJ IDEA 2019.3.4 (Ultimate Edition)
Build #IU-193.6911.18, built on March 17, 2020

Runtime version: 11.0.6+8-b520.43 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6

Bazel plugin: 2020.03.23.0.3

```
Testing started at 1:34 PM ...
/Users/drichelson/bin/bazel coverage --tool_tag=ijwb:IDEA:ultimate --curses=no --color=yes --progress_in_terminal_title=no --combined_report=lcov --runs_per_test=1 --flaky_test_attempts=1 --build_event_binary_file=/var/folders/hk/4yst_rf17wb_1xt9rksfsb9h0000gp/T/intellij-bep-ceedbfee-093c-4858-9f47-a00bfa1c3777 --nobuild_event_binary_file_path_conversion --test_filter=com.salesforce.servicelibs.gater.core.impl.ActivationStrateyImplTest# -- //projects/libs/servicelibs/gater/gater/gater-core-impl:src/test/java/com/salesforce/servicelibs/gater/core/impl/ActivationStrateyImplTest
Running Salesforce build hook script in file: //tools/bazel
Build command: coverage  Time: Mon Apr 20 13:34:23 PDT 2020
Verbose environment data has been logged to /tmp/bazel_env.txt
Verbose process data has been logged to /tmp/bazel_ps.txt
INFO: Invocation ID: c9aae282-44ee-4d4a-aee3-ba048f1224c7
Loading: 
Loading: 0 packages loaded
INFO: Using default value for --instrumentation_filter: "//projects/libs/servicelibs/gater/gater/gater-core-impl[/:]".
INFO: Override the above default with --instrumentation_filter
INFO: Build options --collect_code_coverage, --explicit_java_test_deps, --instrumentation_filter, and 3 more have changed, discarding analysis cache.
Analyzing: target //projects/libs/servicelibs/gater/gater/gater-core-impl:src/test/java/com/salesforce/servicelibs/gater/core/impl/ActivationStrateyImplTest (0 packages loaded, 0 targets configured)
Analyzing: target //projects/libs/servicelibs/gater/gater/gater-core-impl:src/test/java/com/salesforce/servicelibs/gater/core/impl/ActivationStrateyImplTest (72 packages loaded, 1645 targets configured)
Analyzing: target //projects/libs/servicelibs/gater/gater/gater-core-impl:src/test/java/com/salesforce/servicelibs/gater/core/impl/ActivationStrateyImplTest (72 packages loaded, 1645 targets configured)
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:488)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:431)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:212)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:143)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:271)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:130)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:108)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:541)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:204)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:601)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:652)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

**Comment by ulfjack** [comment]
That looks a lot like https://github.com/bazelbuild/bazel/issues/6450, which I fixed in Bazel 2.1.0. Can you try a newer Bazel binary?

**Comment by liucijus** [comment]
Closing, coverage seems to be working
